src/pages/Login/Auth.jsx

```diff
@@ -0,0 +1,57 @@
+
+import { auth , googleProvider } from "../../firebase/Firebase"; // Import auth from your firebase config
+import {createUserWithEmailAndPassword, signInWithPopup, signOut} from "firebase/auth"
+import { useState } from "react"; // Import useState from react
+
+export const Auth = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const signIn = async () => {
+		try {
+			const userCredential = await createUserWithEmailAndPassword(auth, email, password);
+			const user = userCredential.user;
+		} catch (error) {
+			console.log(error);
+		}
+    };
+
+	const signInWithGoogle = async () => {
+		try {
+			const userCredential = await signInWithPopup(auth, googleProvider);
+			const user = userCredential.user;
+		} catch (error) {
+			console.log(error);
```

Comment:
ah, this should add the notifications for wrong auth. 


`
<Alert severity="error">{error}</Alert>
`